openstack/workflow/v2/workflows/results.go

```diff
@@ -53,4 +42,13 @@ type Workflow struct {
 	// Scope is the scope of the workflow.
 	// Values can be "private" or "public".
 	Scope string `json:"scope"`
+
+	// Tags is a list of tags associated to the workflow.
+	Tags []string `json:"tags"`
+
+	// CreatedAt is the creation date of the workflow.
+	CreatedAt gophercloud.JSONRFC3339ZNoTNoZ `json:"created_at"`
```

Comment:
We'll want this to be of type `time.Time` and you'll need to implement a custom `UnmarshalJSON` func. See here for an example: https://github.com/gophercloud/gophercloud/blob/master/docs/FAQ.md#overriding-default-unmarshaljson-method

Or just grep/search for `UnmarshalJSON` in this repository as this happens frequently.